test/e2e/atlas/deployments_local_test.go

```diff
@@ -49,9 +49,6 @@ func splitOutput(cmd *exec.Cmd) (string, string, error) {
 func TestDeploymentsLocal(t *testing.T) {
 	const deploymentName = "test"
 
-	g := newAtlasE2ETestGenerator(t)
```

Comment:
seems we are creating a project just to make sure to list only local deployments, I added an option to filter local only